modules/edw_group/src/NodeGrants.php

```diff
@@ -106,9 +122,35 @@ protected function getNodeMeetingSectionAccessRecords(NodeInterface $node) {
       throw new \InvalidArgumentException();
     }
     $grants = [];
+
+    $contentManagers = $this->userStorage->loadByProperties(['roles' => 'content_manager']);
+    foreach ($contentManagers as $contentManager) {
+      // Content managers can perform all operations on meeting sections regardless of group.
+      $grants[] = [
+        'realm' => static::EDW_REALM_CONTENT_MANAGERS,
+        'gid' => $contentManager->id(),
+        'grant_view' => 1,
+        'grant_update' => 1,
+        'grant_delete' => 1,
+      ];
+    }
+
     $groups = $this->meetingService->getNodeGroups($node, 'view');
     if (empty($groups)) {
-      // If no groups are set, public access view should be permitted.
+      // If no groups are set, private access should be provided for in-session and
+      // production sector.
+      $access = $node->get('field_access')->value;
+      if (in_array($access, ['participants', 'production_sector'])) {
```

Comment:
Production sector is specific to MLF. Here you can invoke a hook and include 'production_sector' later from custom module.
```
$privateRoles = ['participants'];
$this->moduleHandler->alter('private_access_roles', $privateRoles)
```